purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -99,6 +101,18 @@ class Purchases internal constructor(
             purchasesOrchestrator.updatedCustomerInfoListener = value
         }
 
+    /**
+     * Set a value to be able to handle redemptions of RevenueCat web purchases.
+     * You can set to null to destroy the listener.
+     */
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    var redeemWebPurchaseListener: RedeemWebPurchaseListener?
```

Comment:
I wondered about making this a parameter for the SDK configuration… but since this will need to be something that handles the UI for redeeming web purchases, it seemed more flexible to be able to set it later, but lmk what you think!